xapian-letor/api/letor_internal.cc

```diff
@@ -180,6 +180,12 @@ static void read_problem(const char *filename) {
 	}
 	++elements;
 	++prob.l;
+vector<Xapian::RankList>
```

Comment:
This commit looks funky, I think because the one that tidies up the old code around here comes after it. It looks okay in the overall diff, and we're going to collapse these all down once we're happy anyway, so it shouldn't be a problem.